jekyll/_cci2/language-android.md

```diff
@@ -9,20 +9,37 @@ order: 9
 
 *[Tutorials & 2.0 Demo Apps]({{ site.baseurl }}/2.0/tutorials/) > Language Guide: Android*
 
-This guide will help you get started with an Android application on CircleCI.
+This document describes
```

Comment:
I'm curious more than anything about the really short lines - I try to keep markdown to either 80 or 100 characters - looks like this line is about 20 characters - any particular reason to keep it so short?

Reply:
@marcomorain For the Google Cloud SDK bit, I know we note that in the Android Language guide, but are you saying it would be worth noting here in the Google Cloud doc?

And as for the really short lines, those are [semantic linefeeds](http://rhodesmill.org/brandon/2012/one-sentence-per-line/). The intent is to split sentences into chunks so I know when things are getting too long/complex. This makes things much easier to edit at a granular level.